kickstarter/android-oss
# EP-489:  Android QA CTA Clicked (add_ons_continue) Phase 1

# 📲 What

Added three properties to the `add_ons_continue` CTA click event that were not previously being sent: 

```
checkout_add_ons_count_total
checkout_add_ons_count_unique
checkout_add_ons_minimum_usd 
```

# 🤔 Why

Segment integration/QA feedback

# 🛠 How

Added functionality to the `pledgeProperties` method in the `AnalyticEventsUtils.kt` class that sets the three properties listed above. 

# 👀 See
This is the button that triggers the event:
![Screenshot_1618601420](https://user-images.githubusercontent.com/19390326/115075683-77c5d100-9ec9-11eb-8974-232ccfb13b88.png)

# 📋 QA

- Select a project that has add-ons (I used Deck of Dynamic Dungeons)
- Select "Back this project"
- Select a reward
- Add add-ons to and press "Continue with X add-ons"
- In segment, you should see these three properties: 

![Screen Shot 2021-04-16 at 3 13 10 PM](https://user-images.githubusercontent.com/19390326/115075568-44834200-9ec9-11eb-96c1-68b00789c473.png)

# Story 📖

[EP-489:  Android QA CTA Clicked (add_ons_continue) Phase 1](https://kickstarter.atlassian.net/browse/EP-489)
